### DISCUSSION TITLE
Snag hunting!

### DISCUSSION THREAD

**Discussion by callummole** [alan-turing-institute/AutSPACEs#431] [General]
Related to #421, but for the UI. 

Before deployment we want to hunt for any user niggles in the code.

Please run the site locally and comment here any issues you find. To keep things organise will make individual github issues to pool related snags together.

@gedankenstuecke @BrainonSilicon @helendduncan @GeorgiaHCA

**Comment by helen-d-little** [comment]
- [x] Edit (story) button doesn’t work from the view story page (will be solved by #446)
- [ ] Share Stories page `Model`/`Form`: Make the “Suggestions for what could have made your story better” field optional? (forms currently enforces some text before submission)
- [x] My Stories page. Creation date printed format is long and unwieldy (done in #438)
- [x] Moderation: After moderating a story should page redirect to "Moderation" page rather than submission success page
- [ ] Action log: Do we want the PublicExperience model to include an action log of edits etc? This will allow the original creation time to be preserved as well as logs of moderator interaction
- [ ] Set non-public `moderation_status` to empty or something like "private" or "not to be moderated" to highlight that the stories will not be moderated (for example when viewed in the "my stories" page) - will require an update to the model
- [x] Edit button in "view story" page (singular) is in an odd place and not consistent with the "Edit story" page

**Comment by gedankenstuecke** [comment]
Some smaller issues I encountered and also from the meetup notes from the Feb meetup: 

- [x] Mostly relevant in dev environment: "Missing data" leads to "My stories" crashing, see: #439 
- [x] Login button in menu bar not working. It seems the button currently working from the main page, probably because the actual link isn't send to the context in other views (c.f. [here](https://github.com/alan-turing-institute/AutSPACEs/blob/main/server/apps/main/views.py#L28)) 
- [x] “what could have made your story better" when sharing experience is confusing. Potential reword to "what could have made your experience better”. Could also include some sub-headings to better explain the asks. (adressed in #441)
- [x] rename “triggering labels” to “sensitive topics” as "triggering labels" might be hard to understand unless already familiar (adressed in #441)
- [ ] technically implement not allowing people to moderate their own experiences
- [x] found by @callummole: when you change a triggering label during moderation (I changed ‘drugs’ in an example with Robin) the edit  doesn’t get saved properly (the ‘drugs’ label was still ticked).
